core/output/slack.js

```diff
@@ -420,9 +469,13 @@ recMessage = function(event, teamId) {
 			}
 		}
 		else {
-			console.debug('No slack team found matching: ' + teamId);
+			console.debug('slack-> No slack team found matching: ' + teamId);
 		}
 
+		message = message.replace(/&lt;/g, '<');
+		message = message.replace(/&gt;/g, '>');
+		message = message.replace(/&amp;/g, '&');
```

Comment:
Once is for receiving a message and the other is for sending a message, if you check you can see the replacement is reversed